SSA/Projects/InstCombine/InstCombineBase.lean

```diff
@@ -109,11 +113,11 @@ def eval (o : Op)
     | Op.ashr _ => match arg with
       | (arg₁,arg₂) => Bitvec.sshr arg₁ arg₂.toNat
     | Op.const c => c
-    | Op.sub _ => uncurry Bitvec.sub arg
-    | Op.add _ => uncurry Bitvec.add? arg
-    | Op.mul _ => uncurry Bitvec.mul? arg
-    | Op.sdiv _ => uncurry Bitvec.sdiv? arg
-    | Op.udiv _ => uncurry Bitvec.udiv? arg
+    | Op.sub _ => (uncurry Bitvec.sub arg)
+    | Op.add _ => (uncurry Bitvec.add? arg).get!
```

Comment:
I'd rather have a sorries than `get!`s here to be frank

Reply:
@goens sure, SGTM. I'll make the change.

Reply:
BTW, for reference, I am happy with you pushing to this branch and then merging once you're happy with the `get! -> sorry` conversion.